x/ibc/20-transfer/types/packet.go

```diff
@@ -53,3 +60,19 @@ func (pd PacketData) ValidateBasic() sdk.Error {
 	}
 	return nil
 }
+
+// Implements ibc.Packet
+// TODO: need to be hashed(non-unmarshalable) format
+func (pd PacketDataTransfer) GetCommitment() []byte {
```

Comment:
Same comment as above (packets should have `GetData()`, then the IBC keeper should hash, unless there is a particular reason to do otherwise)

Reply:
Misunderstood, will fix it

Reply:
Not as far as I can tell, since it's still here.

Reply:
Still using the name `GetCommitment` but hashing is done inside the channel keeper. It can be renamed. TODO comment has to be removed.